vnc.html

```diff
@@ -171,6 +171,7 @@ <h1 class="noVNC_logo" translate="no"><span>no</span><br>VNC</h1>
                             <option value="off">None</option>
                             <option value="scale">Local Scaling</option>
                             <option value="remote">Remote Resizing</option>
+                            <option value="browser" disabled="" id="noVNC_setting_browser_resize">Resizing browser</option>
```

Comment:
I don't think we need a new setting for this. This should be possible to have active as long as things aren't configured to `"scale"`.